lci_strategic_plugin/src/lci_strategic_plugin.cpp

```diff
@@ -952,22 +825,14 @@ void LCIStrategicPlugin::planWhenAPPROACHING(const cav_srvs::PlanManeuversReques
     ts_params.case_num = CASE_8;
     print_params(ts_params);
   }
+  else if (distance_remaining_to_traffic_light <= desired_distance_to_stop && in_tdb) // Given ET is in TDB, but vehicle is too close to intersection
```

Comment:
I think you must add "ts_params.is_algorithm_successful && ts_params.case_num != TSCase::CASE_8 &&" to the else if condition here as well

Reply:
This is just for debugging purposes, and it maybe better to cover more conditions without restricting to ts_params.is_algorithm_successful && ts_params.case_num != TSCase::CASE_8. As those conditions can be referenced from other log lines.

Reply:
I see. If the handlefailurecase and handlestoppingcase for this edge case is covered by the later logic, then this is fine.